### PR TITLE
PR: Update badges in README

### LINKED ISSUE #254: Update badges in Readme
Now that testing was moved to Github actions, we need to remove the old CI and coverage badges from our Readme and add the new ones.

@jitseniesen, could you take care of that?

### PR DESCRIPTION
**Pull request by jitseniesen**
Fixes #254

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +70,7 @@ or create a issue in this repo. We will be more than glad to answer.
 
 Support us with a monthly donation and help us continue our activities.
 
-[![Backers](https://opencollective.com/spyder/backers.svg)](https://opencollective.com/spyder#support)
+[![Backers](https://opencollective.com/spyder/backers.svg?width=1000)](https://opencollective.com/spyder#support)
```
Comment (ccordoba12) [inline README.md]: Could you change this section to be like the [last one](https://github.com/spyder-ide/spyder-kernels#sponsors) of spyder-kernels' Readme? We're trying to show the same section in all our projects' Readmes.

**Comment by ccordoba12** [comment]
Also, please change in our Readme the minimal Spyder version this plugin depends on.

File: README.md (hunk 1 above)
Comment (jitseniesen) [reply README.md]: Done

**Comment by jitseniesen** [comment]
> Also, please change in our Readme the minimal Spyder version this plugin depends on.

Well spotted!

**Pull request merged**